File: benchmarks/cross-language-v1/tasks/cold-chain-release-gate-v1/public/typescript/candidate.ts
export function releaseAllowed(coreTemperature: number, sealPressure: number): number {
  return coreTemperature >= 2 && coreTemperature <= 8 && sealPressure >= 95 && sealPressure <= 105
    ? 1
    : 0;
}

File: benchmarks/cross-language-v1/tasks/cold-chain-release-gate-v1/public/rust/candidate.rs
pub fn release_allowed(core_temperature: i64, seal_pressure: i64) -> i64 {
    if core_temperature >= 2
        && core_temperature <= 8
        && seal_pressure >= 95
        && seal_pressure <= 105
    {
        1
    } else {
        0
    }
}
